docs/YIP/YIP-0005-session-key-management.md

```diff
@@ -0,0 +1,57 @@
+# 0005 - Session Key Management for ClearSync
+
+## Status
+
+Work in progress
+
+## Context
+
+The adoption of session keys enhances security and operational efficiency.
+By limiting the validity of keys and associating them with specific counterparts, we significantly reduce risks associated with key exposure and unauthorized access.
+
+This proposal introduces a new strategy for key management in the ClearSync project, shifting from signing sub-keys to registering session keys.
+Each Clearport will generate a key pair limited to a default duration of 24 hours, termed a clearing session.
+
+## Decision
+
+### Key Generation and Registration
+
+- **Generation**: Each Clearport generates a key pair valid for a default duration of 24 hours.
+- **Registration**: Participants must register the clearing public key on the adjudicator, specifying the duration, counterpart, and owner (Smart-Account).
+- **Duration**: 24 hours
+- **Counterpart**: Smart-Account
+- **Owner**: Smart-Account
+
+### Key Features
+
+- **Spending Limit**: A spending limit can be added to the clearing key.
+- **Collateral Association**: The collateral is associated with this clearing key.
```

Comment:
What this association is responsible for? 
If its to return collateral the owner's or forced settlement public key is used, isn't it?

Reply:
the collateral that you put in the channel should be associated with your session key and in case of expire - force close